zeek-archiver/CMakeLists.txt

```
@@ -3,4 +3,4 @@ add_executable(zeek-archiver zeek-archiver.cc)
 set_target_properties(zeek-archiver PROPERTIES COMPILE_FLAGS
     "-std=c++17 -Wall -Wno-unused -Werror=vla")
 
-AddAuxInstallTarget(zeek-archiver)
+install(TARGETS zeek-archiver DESTINATION bin)
```

Since the `zeek-archive` target was declared with `add_program` CMake knows it needs to go into `bin/`, so `DESTINATION` is not required (or might even interfere with what was patched into GNUInstallDirs). I'd suggest we drop it here and also for `zeek-cut` below.

```suggestion
install(TARGETS zeek-archiver)
```

---

Ah nice — I did not know that. Done.